index.html

```diff
@@ -25,19 +25,22 @@
             <div class="increase-tempo-content">+</div>
           </button>
         </div>
-        <div class="triangle">
-          <button class="start-stop">
+        <button class="start-stop">
```

Comment:
The idea here is that the start and stop images are _within_ the `<button>`. This way we don't need to modify the contents of the button, just toggle the `stop` (or `stopped`?) class.

Then we display, or not, the images with CSS something like

```css
.start-stop start {
  display: block;
}

.start-stop stop {
  display: none;
}

.start-stop.stopped start {
  display: none;
}

/* etc */
```